frontend/src/app/modules/members/members-page/members-page.component.ts

```diff
@@ -134,23 +138,27 @@ export class MembersPageComponent extends BaseComponent implements OnInit {
     }
 
     deleteMember(memberItem: MemberItem) {
-        this.confirmWindowService.confirm({
-            title: 'Delete member?',
-            message: `Are you sure you wish to delete the <strong>${memberItem.member.user.firstName}`
-                + ` ${memberItem.member.user.lastName} </strong>from the organization?`,
-            acceptButton: { class: 'p-button-primary p-button-outlined' },
-            cancelButton: { class: 'p-button-secondary p-button-outlined' },
-            accept: () => {
-                this.memberService.deleteMember(memberItem.member.id)
-                    .pipe(this.untilThis)
-                    .subscribe(() => {
-                        this.toastNotifications.success('Member deleted');
-                        this.memberItems = this.memberItems.filter(m => m.member.id !== memberItem.member.id);
-                    }, error => {
-                        this.toastNotifications.error(error);
-                    });
-            },
-        });
+        if (memberItem.member.roleId === 1) {
```

Comment:
It's not a good idea rely to roleId. If you need to check this you have 2 options. Do this check on the backend (by name) or here (also by name). Create a separate file with roles name (file with constants)

Reply:
Create exception in backend and check the role name on frontend

Reply:
Update, add exception only backend